### ISSUE TITLE
Logo The Country Network

### ISSUE THREAD

**Issue by Lexzie** [picons/picons-source#339]
Please accept this logo
[TheCountryNetwork_20180312_Lexzie.zip](https://github.com/picons/picons-source/files/1802811/TheCountryNetwork_20180312_Lexzie.zip)

Source logo: https://www.ses.com/north-america/channels
Website TV-station: http://www.tcncountry.com/
Stream URL: http://dcunilive2-lh.akamaihd.net/i/dclive_1@390829/index_150_av-b.m3u8

**Add to snp.index**
```
thecountrynetwork=thecountrynetwork
```

`donkey monkey got issued`

**Comment by mcfly822** [comment]
In better quality:
[thecountrynetwork_mcfly822.zip](https://github.com/picons/picons-source/files/1804414/thecountrynetwork_mcfly822.zip)

**Comment by AbuBaniaz** [comment]
Can we try to make/find better quality logos please? A bit rough on the text edges.

**Comment by mcfly822** [comment]
Yes, I see. I found this logo on the channel's website.
I'll try to find or draw the svg version.

**Comment by mcfly822** [comment]
I could not find in pdf. So I only sent a "trace bitmap" to the original png.
Not perfect, but maybe acceptable. I have no time at this moment to draw the logo line by line.
[tcn_mcfly822.zip](https://github.com/picons/picons-source/files/1806023/tcn_mcfly822.zip)

**Comment by Lexzie** [comment]
Was the 1657x1022 pixels png mcfly found not good enough?

**Comment by AbuBaniaz** [comment]
No, they were not. Did you not see how jagged the text edges were?

I have committed the .svg files

**Comment by mcfly822** [comment]
@AbuBaniaz thanks a lot!
(When I'll have time, I'll improve the svg. Not bad now, but could be better.)

**Comment by Lexzie** [comment]
Well @AbuBaniaz you're probably right.
I thought if you zoom in far enough on a png you'll always see jagged edges.
Of course svg is better!
Great work everyone, thanks!
